packages/core/src/core/pantheon-api.ts

```diff
@@ -10,6 +11,8 @@ export interface ApiRequest {
   query: Record<string, string | string[]> & {
     command: string | string[];
   };
+
+  cookies?: Record<string, string | string[]>;
```

Comment:
These aren't passed in by default in the Nuxt request object so I added a [commit](https://github.com/pantheon-systems/pantheon-content-cloud-sdk/pull/101/commits/d09bb13b1bd669f61e17574ab65fed3470722078) adding forwarding of the cookie object.